react-interactive/src/Layout/VerticalSplit.tsx

```diff
@@ -218,16 +218,22 @@ const VerticalSplit: React.FunctionComponent<IProps> = (props) => {
         let i = 0;
         _.orderBy(elements,(e) => e.Order).forEach((e,index) => {
             const w = Math.floor(scaling * e.Width);
-            if (e.IsDrawer && e.Open !== true)
+            const section = sections[e.Index];
+            const dwr = drawer[e.Index];
+
+            if(section == null)
+
+            if ((e.IsDrawer && e.Open !== true) ||section == null || dwr == null)
```

Comment:
This seems unintentional, since your check gets skipped unless section is null here.